plugins/modules/rulebook_activation.py

```diff
@@ -547,10 +535,21 @@ def endpoint_state(activation: Dict[str, Any], state: str) -> str:
         )
 
     argument_spec.update(AUTH_ARGSPEC)
-
     module = AnsibleModule(argument_spec=argument_spec, supports_check_mode=True)
-    copy_from = module.params.get("copy_from", None)
     restart = module.params.get("restart", None)
+    required_if = []
```

Comment:
@B-Whitt Why we are reintroducing the `required_if` parameter? It was removed to be able to enable/disable activations with only providing name and state, and the logic changed to let the backend return in case of missing fields.

Maybe you probably need to rebase from main first?